docs/cli.md

```diff
@@ -223,16 +223,38 @@ These commands will execute two different project revisions based on the given G
 
 ### Pipeline parameters
 
-Pipeline scripts can use an arbitrary number of parameters that can be overridden using the command line or Nextflow configuration files. Any script parameter can be specified on the command line by prefixing the parameter name with double-dash characters. For example:
+Pipeline scripts can define *parameters* which can be overridden on the command line.
+
+Parameters can be declared in the main script:
+
+```nextflow
+params.alpha = 'default script value'
+
+workflow {
+  println "alpha = ${params.alpha}"
+}
+```
+
+Or in a config file:
+
+```groovy
+params {
+  alpha = 'default config value'
+}
+```
+
+The above parameter can be specified on the command line as `--alpha`:
 
 ```console
-$ nextflow run <pipeline> --foo Hello
+$ nextflow run <pipeline> --alpha Hello
```

Comment:
```suggestion
nextflow run main.nf --alpha Hello
```

Reply:
The `$` is just for consistency with the rest of the page. I wouldn't be opposed to reviewing them in a separate PR. They are useful for showing commands and output together, but aside from that they just get in the way of copying

Reply:
Ahh! Apologies! Agree about a separate PR.